intro-HPC/ch_running_interactive_jobs.tex

```diff
@@ -406,3 +410,20 @@ \subsection{Run your interactive application}
 \begin{prompt}
 fx >> %\textbf{exit}%
 \end{prompt}
+
+
+Please not that you are not running MATLAB on the \hpc, but only on the
+login nodes. So it is not recommended to run large programs this way.
+You are allowed to run X programs on the login nodes for vizualization
```

Comment:
`visualization` (i think aspell has a latex mode. might want to run it over the whole text)
what are "X  programs"?